### PR TITLE
Add Fastlane

### PR DESCRIPTION
**Pull request by mandybess**
## 🔧 changes
- Add fastlane
- Add setup script
- Add signing config to release build script

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Release To The Google Play Store
+
+## Setup
+
+Run the setup script:
+
+```
+> bin/setup
```
Comment (croaky) [inline RELEASING.md]: Drop the `> `? Maybe make it `./bin/setup`? http://stackoverflow.com/questions/6331075/why-do-you-need-dot-slash-before-script-name-to-run-it-in-bash

File: RELEASING.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+```
+
+## Releasing the app
+
+```
+> fastlane android release
```
Comment (croaky) [inline RELEASING.md]: It looks like `>` is standard `PS2` but `PS1` is `$` on bash and `%` on zsh?

http://superuser.com/questions/57575/what-is-the-origin-of-the-unix-dollar-prompt
http://zsh.sourceforge.net/Guide/zshguide02.html#l19

I feel like we don't have a standard prompt style on the blog or for docs. @JoelQ or @gabebw or @mike-burns or @jferris might know. I think I've tended to not put the prompt to make it more copy-pasteable without accidentally copying the prompt. Might even pair with text beforehand like "From your prompt:" or "From your shell:"

File: RELEASING.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+```
+
+See the [fastlane documentation](https://github.com/thoughtbot/tropos-android/blob/master/fastlane/README.md) for more options and details
+
```
Comment (croaky) [inline RELEASING.md]: Extra blank line

File: setup [hunk 4]
```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```
Comment (croaky) [inline setup]: Should this be in `bin` directory?

File: setup [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+# Exit if any subcommand fails
+set -e
+
+# check if keystore exists
+if [ ! -e "app/tropos.keystore" ]; then
+  echo "File \`app/tropos.keystore\` does not exist, this will prevent you from
+  creating release builds." echo "To fix this, download the keystore from
+  thoughtbot 1Password and paste it into \`app/tropos.keystore\`.\n"
```
Comment (croaky) [inline setup]: Does the `echo` mid-line work on line 9?

File: RELEASING.md (hunk 2 above)
Comment (mike-burns) [inline RELEASING.md]: I have this zsh alias that allows me to copy `$` and `%` prompts without problem:

```sh
alias \$=\;
alias \%=\;
```

`%`, `$`, and `#` are commonly understood in the grumpy old unix world to mean "this is to be run at your shell" (`#` means "this is to be run with elevated privileges"). So I'd pick one of those if you're going to show the prompts.

File: setup (hunk 4 above)
Comment (mandybess) [inline setup]: ah yes good catch! 👍

**Pull request merged**